### DISCUSSION TITLE
DWMBlurGlass GUI hangs as soon as it launches.

### DISCUSSION THREAD

**Discussion by dslayah** [Maplespe/DWMBlurGlass#163] [Q&A]
Whenever I open the DWMBlurGlass app, as soon as it opens, it outright locks up and I'll have to force close it through the error reporting thing. I didn't even get to click the install button. I have made sure it's installed outside the Users folder. Any solution would be welcome, I am using Windows 10 22H2.

**Comment by Maplespe** [comment]
Please check to see if your files are complete, and also make sure your DirectX components are complete.
Some settings in the graphics card driver may also cause unexpected problems.

**Comment by dslayah** [comment]
> Please check to see if your files are complete, and also make sure your DirectX components are complete. Some settings in the graphics card driver may also cause unexpected problems.

I have in fact extracted the entire zip to the C:/ directory. What graphic cards settings should I review? I am a laptop user and it has dual GPUs, Nvidia and AMD.

**Comment by Maplespe** [comment]
You can try choosing a different GPU to run on.
Also data/defaultres.dmres in the DWMBlurGlass directory is an important resource file and you need to make sure it exists.

**Comment by UCyborg** [comment]
I believe it has to do with something in the system getting bugged up if it wasn't restarted in a while. It worked cca. 2 weeks ago when I installed it, today I get the same issue reported here. Tried temporarily stopping NVIDIA Display Container Service in services.msc and disabling the graphics driver in Device Manager as well as logging-off and logging-in again, no change. Then I restarted Windows and now it works...until next time.
